build_tools/circle/build_doc.sh

```diff
@@ -51,7 +51,7 @@ get_build_type() {
     fi
     git_range="origin/master...$CIRCLE_SHA1"
     git fetch origin master >&2 || (echo QUICK BUILD: failed to get changed filenames for $git_range; return)
-    filenames=$(git diff --name-only $git_range)
+    filenames=$(git diff --name-only $git_range -- `find . -name plot_*`)
```

Comment:
According to the logs, maybe we should filter these files at changed_examples below?

Reply:
yeah, didn't think it's important, but now I applied the same filter there.

Reply:
This seems inconsistent with our output
```
filenames=XXX
if [ -z "$filenames" ]
    echo QUICK BUILD: no changed filenames for $git_range
    return
changed_examples=XXX
if [[ -n "$changed_examples" ]]
    echo BUILD: detected examples/ filename modified in $git_range: $changed_examples
    return
echo QUICK BUILD: no examples/ filename modified in $git_range:
```
so seems that filenames include all the files, not only examples.